### ISSUE TITLE
AGOL Edits not reliabally reflected in Collector

### ISSUE THREAD

**Issue by FsenyahPHE** [doug-m/CollectorAuroraTesting#128]
### Issue

When deleting features from AGOL while working online, the edits don’t show in the App unless it is closed or a new map is opened in the interim, and in some instances these workarounds still don’t cause the deletions to show in the App even after a refresh, close and reopen. Note, this is not a common workflow of ourse currently but it could be in the future and would certainly cause issues.  Working the other way around (i.e. delete in the App) seems to work a lot more relaibally with edits appearing immediately when the web map is panned or refreshed.

### Expected Behavior
Edits made in the Web Map should be reliabally viewable in the App to reduce potential errors and duplication of effort.

### Steps to Reproduce

-Open a map in the App
-Make edits to map features in AGOL
-Edits in AGOL aren't immediately honoured in the App, open, close and refresh also don't always force edits to be reflected

### Device Information
IPhone 7
Version 11.4

**Comment by doug-m** [comment]
@FsenyahPHE do you have a refresh interval specified on your layer that the feature was updated/deleted?

**Comment by FsenyahPHE** [comment]
@doug-m No, thisisn't a feature I have used to date actually.  It currrently refreshes every 0 minutes, presumbly the default setting.

**Comment by doug-m** [comment]
@FsenyahPHE thanks, sounds like it is not refreshing.  Would you be willing to specify one (you can go as low as 0.1 minutes when setting the refresh interval) and try this scenario again?

**Comment by FsenyahPHE** [comment]
Sure, I'll do that and let you know the outcome.

**Comment by FsenyahPHE** [comment]
@doug-m Just tested with a 0.1 minute interval and it seems to work as expected.
